examples/Modal.js

```diff
@@ -1,10 +1,47 @@
-import React from 'react';
+import React, {Component} from 'react';
 import Modal from '../src/Modal';
 import Button from '../src/Button';
 
-export default
+class ModalForm extends Component {
+  constructor(){
+    super()
+    this.state = {
+      firstName: "",
+      lastName: "",
+      email: ""
+    }
+    
+    handleChange = (event) => {
+      var newState = {};
+      newState[event.target.id] = event.target.value;
+      this.setState(
+        newState
+        );
+    }
+    
+    handleSubmit = (e) => {
+      e.preventDefault();
+      //your function
+    }
+    
+    render(){
+      return(
 <Modal
-  header='Modal Header'
-  trigger={<Button>MODAL</Button>}>
-  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum</p>
-</Modal>;
+	header='Modal Header'
+	modalOptions={{ dismissible: false, ready: () => { console.log('ready'); } }}
+	fixedFooter
+  trigger={<Button className="mainBtn">MODAL</Button>}
+  >
+    <Row>
+     <h5>First Name</h5>
+      <Input type="text" id="firstName" value={this.state.firstName} onChange={this.handleChange}/>
+	   <h5>Last Name</h5>
+      <Input type="text" id="lastName" value={this.state.firstName} onChange={this.handleChange}/>
+	   <h5>email</h5>
+      <Input type="text" id="email" value={this.state.email} onChange={this.handleChange} />  
+    <Button id="submitButton" type="submit" className="mainBtn" onClick={this.handleSubmit}>Submit</Button>
+
+  <p>Press submit</p>
+</Modal>
```

Comment:
This is syntactically not correct there is not closing parethesis and bracket. Please try it on your local. Simply go to the `docs/` folder and run `npm i && npm start`